src/main/java/jenkins/plugins/http_request/HttpRequest.java

```diff
@@ -175,17 +197,31 @@ public String getValidResponseContent() {
         return validResponseContent;
     }
 
-    @Override
-    public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListener listener) throws InterruptedException, IOException {
-        defineDefaultConfigurations();
+    @Deprecated
+    public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListener listener)
+    throws InterruptedException, IOException {
+        perform(build, build.getWorkspace(), launcher, listener);
+        return true;
+    }
```

Comment:
No need to retain this overload. The `super` implementation will call the new one automatically.